youtube_dl/extractor/drtuber.py

```diff
@@ -44,8 +44,23 @@ def _real_extract(self, url):
         webpage = self._download_webpage(
             'http://www.drtuber.com/video/%s' % video_id, display_id)
 
-        video_url = self._html_search_regex(
-            r'<source src="([^"]+)"', webpage, 'video URL')
+        video_data = self._download_json('http://www.drtuber.com/player_config_json/', video_id, query={
+            'vid': video_id,
+            'embed': 0,
+            'aid': 0,
+            'domain_id': 0,
+        }, fatal=True)
+
+        formats = []
+        for video_url_key in video_data.get('files', {}):
```

Comment:
What's the point of `.get('files', {})`?
Use `for k,v in data.items()` instead.